bioio/tests/conftest.py

```diff
@@ -90,3 +92,9 @@ def __exit__(
         subprocess.check_call(
             [sys.executable, "-m", "pip", "uninstall", "-y", self.package_name]
         )
+
+
+@pytest.fixture
+def dummy_plugin() -> typing.Generator[str, None, None]:
+    with InstallPackage(package_path=DUMMY_PLUGIN_PATH, package_name=DUMMY_PLUGIN_NAME):
+        yield DUMMY_PLUGIN_NAME
```

Comment:
Unrelated to main changeset, this is just to make the tests faster by only installing the plugin once since they seemed slow